src/components/Header/Header.stories.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import GlobalStyles from '../../GlobalStyles'
+import { addDecorator } from '@storybook/react'
+import Header from './Header'
+
+addDecorator((storyFn) => (
+  <>
+    <GlobalStyles />
+    {storyFn()}
+  </>
+))
+
+export default {
+  title: 'Header',
+  component: Header,
+}
+
+export const withTitle = () => <Header />
```

Comment:
Falls du vor hast deinem Header auch mal einen anderen Inhalt zu geben, kannst du hier dem Header-tag auch ein `value="digitalnatur"` prop übergeben.

Reply:
Hab ich direkt in meine ToDoListe aufgenommen :-)